e2e/config/host.docker.internal/zitadel.yaml

```diff
@@ -3,7 +3,7 @@ ExternalSecure: false
 
 Database:
   cockroach:
-    Host:  db
+    Host:  host.docker.internal
```

Comment:
This makes the config reusable with an out-of-docker zitadel process and an /etc/hosts entry

Reply:
Please add the comment to the file

Reply:
done